app/components/ui/SwapInput.tsx

```diff
@@ -9,10 +9,11 @@ import { TokenLogo } from '../TokenLogo';
 interface SwapInputProps extends InputHTMLAttributes<HTMLInputElement> {
   title: string;
   selectedToken: Token | Outcome | null;
-  onTokenClick?: (outcome?: Outcome) => void;
-  tokenList: Array<Outcome> | null;
+  onTokenClick?: (asset?: Token | Outcome) => void;
+  tokenList: Array<Token | Outcome> | null;
 }
 
+//TODO create 2 different instances, 1 for token and other for outcomes
```

Comment:
should we handle this comment on this PR?

Reply:
We should handle this in a new task. Will delete the comment and create a task.